Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1715,5 +1715,13 @@ bool WebChromeClient::isUsingUISideCompositing() const
 #endif
 }
 
+bool WebChromeClient::isInStableState() const
+{
+#if PLATFORM(IOS_FAMILY)
+    return m_page.isInStableState();
+#else
```

Comment:
This is a little surprising -- surely macOS has snapping and rubber-banding that might leave the client in an unstable state. Or is this stability about something that truly doesn't apply for macOS?

Reply:
Ah, yeah — I think ideally, this would be sensibly implemented on macOS as well, but implementing that seemed non-trivial, and in the spirit of keeping the fix targeted I decided to hold off on that part. Definitely warrants a FIXME, though.